karma.conf.js

```diff
@@ -60,6 +60,7 @@ module.exports = function (config) {
     files: [
       'node_modules/babel-polyfill/dist/polyfill.js',
       'node_modules/js-data/dist/js-data.js',
+      'node_modules/js-data-adapter/dist/js-data-adapter.js',
```

Comment:
This shouldn't need to be loaded, as `js-data-adapter` is bundled into `dist/js-data-firebase.js`